src/app/options/options.component.ts

```diff
@@ -33,8 +33,24 @@ export class OptionsComponent {
       });
     }),
   );
+  additionalOptions: String = this.service.data.additionalOptions;
+  submit$: Observable<boolean> = of(false);
```

Comment:
can you rename this to "canSubmit" so it's more readable.

Reply:
Yes but will be "submitting" since it's true when submitting not when you can submit.